CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ the [releases page](https://github.com/Consensys/teku/releases).
 
 ### Breaking Changes
 - By default, Teku won't allow syncing from genesis, users should use `--checkpoint-sync-url` when starting a new node. It is possible to revert back to the previous behaviour using the flag `--ignore-weak-subjectivity-period-enabled`.
+- Teku will fail to start if the Validator API cannot read the password file used for authorization.
```

Comment:
Maybe add something like 'Previously, if the validator-api was enabled and the password file was not available, the validator would continue to startup without the validator-api` or something...